network-api/networkapi/wagtailpages/templates/wagtailpages/tags/picture_ratios.html

```diff
@@ -0,0 +1,25 @@
+{% load wagtailimages_tags %}
+
+<picture class="{{class_names}}">
+    {# High quality large img #}
+    {% image unoptimized_image fill-540x304-c0 format-jpeg as image_large_hq %}
+    {# Low quality large img #}
+    {% image unoptimized_image fill-540x304-c0 format-jpeg as image_large_lq %}
+    <source media="(min-width: 992px)" srcset="{{ image_large_lq.url }} 1x, {{ image_large_hq.url }} 2x" type="image/jpeg" alt="{{ image_large_hq.alt }}">
```

Comment:
we're probably going to have to either leave off the `type`, or make sure we template in the mimetype, because many of our images are png or svg, in addition to jpg.

Reply:
I was mistaken, wagtail turns everything into .jpg when it runs tags.